EasyPost.Tests/ModelsTests/UserTest.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EasyPost.Models.API;
+using EasyPost.Tests._Utilities;
+using EasyPost.Tests._Utilities.Annotations;
+using EasyPost.Utilities.Annotations;
+using Xunit;
+
+namespace EasyPost.Tests.ModelsTests
+{
+    public class UserTests : UnitTest
+    {
+        public UserTests() : base("user", TestUtils.ApiKey.Production) =>
+            CleanupFunction = async id =>
+            {
+                try
+                {
+                    User retrievedUser = await Client.User.Retrieve(id);
+                    await retrievedUser.Delete();
+                    return true;
+                }
+                catch
+                {
+                    // trying to delete something that doesn't exist, pass
+                    return false;
```

Comment:
Question: Why are we returning a bool here? Is it necessary?

Reply:
This is a guard clause.